docs/standard/numerics.md

```diff
@@ -33,44 +33,58 @@ The .NET Framework supports the standard numeric integral and floating-point pri
+
+You can also work with the individual bits in an integer value by using the <xref:System.BitConverter?displayProperty=nameWithType> class.  
+
+> [!NOTE]  
+> The unsigned integral types are not CLS-compliant. For more information, see [Language Independence and Language-Independent Components](language-independence-and-language-independent-components.md).
+
+## BigInteger
+
+The <xref:System.Numerics.BigInteger?displayProperty=nameWithType> structure is an immutable type that represents an arbitrarily large integer whose value in theory has no upper or lower bounds. The methods of the <xref:System.Numerics.BigInteger> type closely parallel those of the other integral types.
   
-## SIMD-enabled vector types  
- The <xref:System.Numerics> namespace includes a set of SIMD-enabled vector types for the .NET Framework. SIMD (Single Instruction Multiple Data operations) allows some operations to be parallelized at the hardware level, which results in huge performance improvements in mathematical, scientific, and graphics apps that perform computations over vectors.  
+## Floating-point types
+
+.NET includes three primitive floating-point types, which are listed in the following table:
   
- The SIMD-enabled vector types in the .NET Framework include the following: .  In addition, System.Numerics.Vectors includes a Plane type and a Quaternion type.  
+|Type|Size (in bytes)|Approximate range|Precision|  
+|----------|--------|---------------------|--------------------|  
+|<xref:System.Single?displayProperty=nameWithType>|4|±1.5 x 10<sup>−45</sup> to ±3.4 x 10<sup>38</sup>|7 digits|  
+|<xref:System.Double?displayProperty=nameWithType>|8|±5.0 × 10<sup>−324</sup> to ±1.7 × 10<sup>308</sup>|15-16 digits|  
+|<xref:System.Decimal?displayProperty=nameWithType>|16|±1.0 x 10<sup>-28</sup> to ±7.9228 x 10<sup>28</sup>|28-29 digits|  
   
--   <xref:System.Numerics.Vector2>,  <xref:System.Numerics.Vector3>, and  <xref:System.Numerics.Vector4> types, which are 2-, 3-, and 4-dimensional vectors of type <xref:System.Single>.  
+Each floating-point type supports standard arithmetic operators. The <xref:System.Math?displayProperty=nameWithType> class provides methods for a broader set of mathematical functions. .NET Core 2.0 and later includes the <xref:System.MathF?displayProperty=nameWithType> class that provides methods which accept arguments of the <xref:System.Single> type.
+
+You can also work with the individual bits in <xref:System.Double> and <xref:System.Single> values by using the <xref:System.BitConverter?displayProperty=nameWithType> class. The <xref:System.Decimal?displayProperty=nameWithType> structure has its own methods, <xref:System.Decimal.GetBits%2A?displayProperty=nameWithType> and <xref:System.Decimal.%23ctor%28System.Int32%5B%5D%29?displayProperty=nameWithType>, for working with a decimal value's individual bits, as well as its own set of methods for performing some additional mathematical operations.
```

Comment:
Should we have a comment about the methods on `System.Single` and `System.Double` that allow checking for special values or categories (i.e. `IsNaN`, `IsFinite`, etc)